documents/motions/04.2019.1 - Board of Directors.md

```diff
@@ -23,11 +23,12 @@ The WCA Board of Directors (or simply WCA Board) is the leadership team of the W
    2. To approve amendments to the Regulations. Such rights must be exercised in accordance with [WCA Motion : Amendments of Regulations](wca{documents/motions/14.2019.1 - Amendments of Regulations.pdf}).
    3. To establish and remove Committees or change their duties. Such rights must be exercised in accordance with the Motion relative to every Committee.
    4. To instate and remove Senior Delegates. Such rights must be exercised in accordance with [WCA Motion : Senior Delegates](wca{documents/motions/09.2019.1 - Senior Delegates.pdf}).
-   5. To suspend or take other sanctions against Registered Speedcubers and to reinstate Registered Speedcubers that have been suspended. Such rights must be exercised in accordance with [WCA Motion : Suspensions and other Sanctions](wca{documents/motions/15.2019.1 - Suspensions and other Sanctions.pdf}).
-   6. To approve the annual budget proposal of the WCA Financial Committee in accordance with [WCA Motion : Finances](wca{documents/motions/11.2019.1 - Finances.pdf}).
-   7. To approve and announce WCA Competitions.
-   8. To accept or not accept results of WCA Competitions as official, according to the procedures in the Regulations.
-   9. To overrule any decision under control of the WCA, as long as the overruling complies with the WCA Bylaws, their Motions, and the Regulations.
+   5. To instate and remove Regional Delegates. Such rights must be exercised in accordance with [WCA Motion : Regional Delegates](wca{documents/motions/09.2019.2 - Regional Delegates.pdf}).
+   6. To suspend or take other sanctions against Registered Speedcubers and to reinstate Registered Speedcubers that have been suspended. Such rights must be exercised in accordance with [WCA Motion : Suspensions and other Sanctions](wca{documents/motions/15.2019.1 - Suspensions and other Sanctions.pdf}).
+   7. To approve the annual budget proposal of the WCA Financial Committee in accordance with [WCA Motion : Finances](wca{documents/motions/11.2019.1 - Finances.pdf}).
+   8. To approve and announce WCA Competitions.
```

Comment:
This is currently a Right and Duty of the WCAT per the WCAT Motion. Is there any reason it is specifically called out as a right of the Board as well (other than historical)? Wouldn't this be covered by the Board's right to overrule decisions under the control of the WCA?

Reply:
I think it's harmless to leave this as a right--I think it's most important that this is not a duty of the WCA Board.

Reply:
I think it's harmless to keep it but also not worth it. As this is just vestigial, I would remove it.

Reply:
OK, I'm fine with its removal.

Reply:
Since there is no practical use for this anymore, I would also remove it to keep it consistent.

Reply:
I have already included this in a `Staff review` commit that I'll be pushing live later.